### PR TITLE
Pr update to studio v2

### PR DESCRIPTION
**Pull request by MarqRazz**
This PR updates this site config to be compatible with MoveIt Studio V2.0

### CODE REVIEW

File: ur5e_linear_actuator_base_config/objectives/pick_place_ctb.xml [hunk 1]
```diff
@@ -10,10 +10,11 @@
         <Action ID="ModifyObjectInPlanningScene" object="{cuboid_object}" apply_planning_scene_service="/apply_planning_scene"/>
 
         <!-- Set up an MTC task to pick the cuboid and then plan the task. -->
-        <Action ID="InitializeMTCTask" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" task="{task}"/>
+        <Action ID="InitializeMTCTask" task_id="pick_place_ctb" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" task="{task}"/>
         <Action ID="SetupMTCCurrentState" task="{task}"/>
         <Action ID="PickAndPlaceCTB" cuboid_object="{cuboid_object}" task="{task}" parameters="{parameters}"/>
         <Action ID="PlanMTCTask" solution="{solution}" task="{task}"/>
+        <!-- <Action ID="ExecuteMTCTask" solution="{solution}"/> -->
```
Comment (EzraBrooks) [inline ur5e_linear_actuator_base_config/objectives/pick_place_ctb.xml:17]: if this is commented out, will this execute?

**Review by EzraBrooks — COMMENTED** [review]
I appreciate the structural comments in the objective XML!

How should I test this PR?

**Comment by MarqRazz** [comment]
Follow the setup instructions in the [README](https://github.com/PickNikRobotics/robot_on_rails/tree/pr-update_to_studio_v2#readme) and then make sure the robot can move to known poses. If you want to try out the Pick and Place CTB behavior you have to visualize the plan in Rviz.

**Comment by kineticsystem** [comment]
Some notes to test it without Docker.
To test it locally checkout and build this PR. Remember to source `install/setup.bash` in the main moveit_studio repo before building it. Once built, copy the folder `vention_mo_lm_039_2295_description` into `moveit_studio_web/public/`
Before running each command (agent, bridge, api, web) remember to source the two repositories.

**Comment by kineticsystem** [comment]
I see two "Home" endpoints:
image
I recreated the `.config/moveit_studio` folder from scratch.

File: ur5e_linear_actuator_base_config/launch/hardware.launch.py [hunk 2]
```diff
@@ -1,15 +1,44 @@
 # -*- coding: utf-8 -*-
-# Copyright 2023 PickNik Inc.
+# Copyright 2021 PickNik Inc.
```
Comment (kineticsystem) [inline ur5e_linear_actuator_base_config/launch/hardware.launch.py:2]:
```suggestion
# Copyright 2023 PickNik Inc.
```

**Review by kineticsystem — APPROVED** [review]
Built and tested locally. Working fine.

**Pull request merged**